### PR TITLE
Databases week3

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: databases/week3/mealSharing.sql [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+WHERE title LIKE 'Rød grød med%';
+
+--Get meals that has been created between two dates
+SELECT * FROM meal
+WHERE created_date BETWEEN '2022-05-22' AND '2022-07-22';
+
+--Get only specific number of meals fx return only 5 meals
+SELECT * FROM meal
+LIMIT 2;
+SELECT * FROM review;
```
Comment (andrii1) [inline databases/week3/mealSharing.sql]: SELECT * FROM review - should it be here?

File: databases/week3/mealSharing.sql [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+--Get the meals that have good reviews
+SELECT meal.title, review.stars AS starts
```
Comment (andrii1) [inline databases/week3/mealSharing.sql]: little typo AS stars

File: databases/week3/mealSharing.sql [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+SELECT *
+FROM reservation
+WHERE meal_id = 1
+ORDER BY created_date DESC;
+
+--Sort all meals by average number of stars in the reviews
+SELECT meal.title, SUM(review.stars) / COUNT(review.id) AS average FROM meal
+JOIN review ON review.meal_id = meal.id
+GROUP BY meal.title
+ORDER BY average;
```
Comment (andrii1) [inline databases/week3/mealSharing.sql:148]: Cool! I did this with AVG function
SELECT
  meal.title,
  AVG(review.stars)
FROM
  meal
  JOIN review ON review.meal_id = meal.id
GROUP BY
  meal.title
ORDER by
  AVG(review.stars) ASC

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls implement Andrii's feedback as requested. Thx

File: databases/week3/mealSharing.sql (hunk 2 above)
Comment (27ManishaPatel) [reply databases/week3/mealSharing.sql]: Yes true typo error 🙈

File: databases/week3/mealSharing.sql (hunk 3 above)
Comment (27ManishaPatel) [reply databases/week3/mealSharing.sql:148]: yes we can do like this too even its better 👍🏻

**Pull request merged**